src/seed/seedSubjects.js

```
@@ -0,0 +1,46 @@
+
+const SeedSubjects = {
+  createSubjects: async () => {
+    try {
+      const categorySubjectsMap = {
+        'Design': ['UX/UI Design', 'Graphic Design', 'Motion Design'],
+        'Mathematics': ['Algebra', 'Geometry', 'Probability Theory'],
+        'Programming': ['Frontend Development', 'Backend Development', 'Mobile Development']
+      };
+
+      for (const [categoryName, subjects] of Object.entries(categorySubjectsMap)) {
+        const category = await Category.findOne({ name: categoryName });
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)